data-otservbr-global/scripts/actions/farmine/oberon_lever.lua

```diff
@@ -35,6 +35,14 @@ function oberonLever.onUse(creature, item, fromPosition, target, toPosition, isH
 			players[#players+1] = player1
 		end
 		for i, player in ipairs(players) do
+			if player and player:isPlayer() then
+					if player:getStorageValue(setting.storage) > os.time() then
+							player:getPosition():sendMagicEffect(CONST_ME_POFF)
+							player:say('You are still exhausted from your last battle.')
+							return true
+					end
+					table.insert(playersTable, player:getId())
+			end			
```

Comment:
```suggestion
			if player and player:isPlayer() then
					if player:getStorageValue(setting.storage) > os.time() then
							player:getPosition():sendMagicEffect(CONST_ME_POFF)
							player:say('You are still exhausted from your last battle.')
							return true
					end
			end			
```

Tem alguns problemas aqui, primeiramente, o "playersTable" não existe no script. Também me parece que a lógica está errada para a forma que o script funciona, provavelmente deveria rodar numa iteração a parte e antes de criar os monstros.

Reply:
Entendi, acho que faz sentindo, caso for mudar, fazer igual murilo falou, utilizar aquele padrão de boss.